### PR TITLE
Categorization

### PR DESCRIPTION
**Pull request by drfarid**
make sure everything is kosher

### CODE REVIEW

File: ideas-conference-planner/main.js [hunk 1]
```diff
@@ -363,6 +363,10 @@ ipc.on('query-presentations', function(event, arg) {
   event.returnValue = queryPresentations(event);
 });
 
+ipc.on('get-category', function(event, arg) {
```
Comment (mehalter) [inline ideas-conference-planner/main.js]: Is this a mistake? Having a blank function and then the exact same `ipc.on` call right below with information?

Comment (drfarid) [reply ideas-conference-planner/main.js]: the value is set when the dropdown is changed.  You're right, i took it out.

Comment (drfarid) [reply ideas-conference-planner/main.js]: the first category is supposed to be an empty space, it isn't possible to access it. 
We never got to the editing of the categories.

Comment (mehalter) [reply ideas-conference-planner/main.js]: If the first category is supposed to be an empty space, then it doesn't work for me. I have a single category in the database and cannot set any categories to anything. I don't have an empty space in the drop down. It's just the single category. And nothing is every set.

**Pull request closed without merging**